src/FormidableValidations.res

```diff
@@ -27,26 +27,36 @@ module Validator = {
   type t<'values, 'value, 'error> = Args.t<'values, 'value, 'error> => Value.t<'value, 'error>
 }
 
-module Pair = {
-  type t<'values, 'value, 'error> = (list<string>, Validator.t<'values, 'value, 'error>)
+module Description = {
```

Comment:
It's not a pair anymore ^^

Reply:
I like Record more, since like you said it is clearer with the label

Reply:
Thank you for the feedback 😄